kuma/spam/tests/test_akismet.py

```diff
@@ -0,0 +1,151 @@
+from django.test import SimpleTestCase
+from constance.test import override_config
+
+import requests
+import responses
```

Comment:
It is! Basically just a convenience wrapper around mock, but useful nevertheless